spec/policy/headers/headers_spec.lua

```diff
@@ -1,6 +1,12 @@
 local HeadersPolicy = require('apicast.policy.headers')
+local ngx_variable = require 'apicast.policy.ngx_variable'
 
 describe('Headers policy', function()
+  before_each(function()
+    -- avoid stubbing all the ngx.var.* and ngx.req.* in the available context
```

Comment:
Less than ideal, but without this, all the calls to ngx.var.* and ngx.req.* will raise an "API disabled" error.